.dir-locals.el

```diff
@@ -1 +1 @@
-((nil . ((cider-clojure-cli-global-options . "-C:test -R:test"))))
```

Comment:
This change is not directly relevant to this PR but simply a consequence of the preferred Clojure CLI way of handling the thing to be expressed here changing.